lexicons/app/bsky/feed/gate.json

```diff
@@ -0,0 +1,46 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.gate",
+  "defs": {
+    "main": {
+      "type": "record",
+      "key": "tid",
```

Comment:
also think it's wroth adding a note about the rkey constraint here (being the same as the post)

maybe in `description` or something